unison-src/transcripts/alias-many.output.md

```diff
@@ -701,7 +701,7 @@ Let's try it!
   511. Universal.>= : a -> a -> Boolean
   512. Universal.compare : a -> a -> Int
   513. Universal.murmurHash : a -> Nat
-  514. unsafe.coerceAbilities : (a ->{e1} b) -> a ->{e2} b
+  514. unsafe.coerceAbilities : (a ->{e1} b) -> a -> b
```

Comment:
Interesting quirk. The `{e2}` is there, but since it's in strictly positive position, it's elided during printing.